server/database/entities/Example.ts

```diff
@@ -5,10 +5,10 @@ import { Base } from './Base'
 @Entity()
 export class Example extends Base {
   @Column('text')
-    description!: string
+  description!: string
```

Comment:
Were these and the changes in `server/database/entities/Base.ts` changed by the linter? Was this due to the version bump or did they not get linted properly before?

Reply:
This was due a package being bumped, namely @antfu's eslint rules: https://github.com/antfu/eslint-config/releases/tag/v0.27.0

See how they improved the decorator formatting. It was something that always annoyed me and now it's resolved, which is great 🎊